pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -196,7 +196,7 @@ def _run_early_stopping_check(self, trainer: "pl.Trainer") -> None:
         ):  # short circuit if metric not present
             return
 
-        current = logs.get(self.monitor)
+        current = logs[self.monitor].squeeze()
         should_stop, reason = self._evaluate_stopping_criteria(current)
```

Comment:
this is sus
mypy did not detect that `_evaluate_stopping_criteria` had current as not Optional even though .get() could return None. 
Is it because CI does not run with torch installed?

Reply:
I guess, it probably evaluates `torch.Tensor` as `Any`